scripts/updateFinancialSummaryData.js

```diff
@@ -0,0 +1,44 @@
+const moment = require('moment');
+const fetch = require('node-fetch');
+const { getAllSites } = require('../airtable/request');
+
+require('dotenv').config()
+
+
+const SERVER_URL = process.env.SERVER_URL;
+
+// This script is to be run once a day by Heroku Scheduler
```

Comment:
I'm assuming the Heroku Scheduler is run from the Heroku portal and there isn't some cron job configuration that I should be looking for, right?